README.md

```diff
@@ -164,6 +170,34 @@ schedule_immediate_kill= true
     headers = ["header1Key:header1Value","header2Key:header2/Value"]
 ```
 
+#### Placeholders
+
+The message supports the following placeholders:
+* `{$name}`: victim's name
+* `{$kind}`: victim's kind
+* `{$namespace}`: victim's namespace
+* `{$timestamp}`: attack's time from Unix epoch in milliseconds
+* `{$date}`: attack's date
+* `{$error}`: result's error, if any
+
+```json
+  message: '{
+            "what": "Kube-monkey attack of {$name} in {$namespace}",
+            "who": "{$name}",
+            "when": {$timestamp}
+           }'
+```
+
+The header supports a special placeholder to retrieve the value of an environment variable.
+This is useful when calling an API that has a protected endpoint.
+A typical scenario will be to pass an API token to the Kube-monkey container, this token is stored in a Kubernetes Secret and you want to pass it via an environment variable.
+
+```json
+headers = ["api-key:{$env:API_TOKEN}", "Content-Type:application/json"]
+```
+
+`{$env:API_TOKEN}` will be replaced by the environment variable `API_TOKEN` value.
```

Comment:
What about also describing what happens if the env var is not found?

Reply:
Added a note in doc about that with a57ee56